controller.py

```diff
@@ -347,6 +348,15 @@ def edit_registration(self):
         library.registry_stage = registry_stage
         return Response(unicode(library.internal_urn), 200)
 
+    def add_or_edit_pls_id(self):
+        uuid = flask.request.form.get("uuid")
+        library = self.library_for_request(uuid)
+        if isinstance(library, ProblemDetail):
+            return library
```

Comment:
This bit needs a test -- if you pass in an invalid UUID to the controller you get a ProblemDetail.